<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WifiConnectForm</class>
 <widget class="QWidget" name="WifiConnectForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>240</height>
   </rect>
  </property>
  <widget class="QWidget" name="settingsHead" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>320</width>
     <height>40</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <widget class="QLabel" name="settingLabel">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>320</width>
      <height>40</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>WiFi</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="back">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>50</width>
      <height>40</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="icons.qrc">
      <normaloff>:/settings/icons/arrow_left_white.svg</normaloff>:/settings/icons/arrow_left_white.svg</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>18</width>
      <height>18</height>
     </size>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="wifi_settings" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>40</y>
     <width>320</width>
     <height>90</height>
    </rect>
   </property>
   <widget class="QLabel" name="ssid">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>10</y>
      <width>180</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="connect">
    <property name="geometry">
     <rect>
      <x>195</x>
      <y>10</y>
      <width>110</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Connect</string>
    </property>
   </widget>
   <widget class="QPushButton" name="visible">
    <property name="geometry">
     <rect>
      <x>265</x>
      <y>50</y>
      <width>40</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="icons.qrc">
      <normaloff>:/settings/icons/visible_icon.svg</normaloff>:/settings/icons/visible_icon.svg</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>25</width>
      <height>25</height>
     </size>
    </property>
   </widget>
   <widget class="QLineEdit" name="password">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>50</y>
      <width>250</width>
      <height>30</height>
     </rect>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="input" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>130</y>
     <width>320</width>
     <height>110</height>
    </rect>
   </property>
   <widget class="QPushButton" name="backwardsLower">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>10</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="icons.qrc">
      <normaloff>:/settings/icons/arrow_left_white.svg</normaloff>:/settings/icons/arrow_left_white.svg</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="lowerCase">
    <property name="geometry">
     <rect>
      <x>45</x>
      <y>10</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string notr="true">a</string>
    </property>
   </widget>
   <widget class="QPushButton" name="forwardsLower">
    <property name="geometry">
     <rect>
      <x>85</x>
      <y>10</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="icons.qrc">
      <normaloff>:/settings/icons/arrow_right_white.svg</normaloff>:/settings/icons/arrow_right_white.svg</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="upperCase">
    <property name="geometry">
     <rect>
      <x>45</x>
      <y>60</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string notr="true">A</string>
    </property>
   </widget>
   <widget class="QPushButton" name="numbers">
    <property name="geometry">
     <rect>
      <x>175</x>
      <y>10</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string notr="true">0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="symbols">
    <property name="geometry">
     <rect>
      <x>175</x>
      <y>60</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string notr="true">!</string>
    </property>
    <property name="iconSize">
     <size>
      <width>22</width>
      <height>22</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="backwardsNumbers">
    <property name="geometry">
     <rect>
      <x>135</x>
      <y>10</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="icons.qrc">
      <normaloff>:/settings/icons/arrow_left_white.svg</normaloff>:/settings/icons/arrow_left_white.svg</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="backwardsUpper">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>60</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="icons.qrc">
      <normaloff>:/settings/icons/arrow_left_white.svg</normaloff>:/settings/icons/arrow_left_white.svg</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="backwardsSymbols">
    <property name="geometry">
     <rect>
      <x>135</x>
      <y>60</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="icons.qrc">
      <normaloff>:/settings/icons/arrow_left_white.svg</normaloff>:/settings/icons/arrow_left_white.svg</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="forwardsNumbers">
    <property name="geometry">
     <rect>
      <x>215</x>
      <y>10</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="icons.qrc">
      <normaloff>:/settings/icons/arrow_right_white.svg</normaloff>:/settings/icons/arrow_right_white.svg</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="forwardsUpper">
    <property name="geometry">
     <rect>
      <x>85</x>
      <y>60</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="icons.qrc">
      <normaloff>:/settings/icons/arrow_right_white.svg</normaloff>:/settings/icons/arrow_right_white.svg</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="forwardsSymbols">
    <property name="geometry">
     <rect>
      <x>215</x>
      <y>60</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="icons.qrc">
      <normaloff>:/settings/icons/arrow_right_white.svg</normaloff>:/settings/icons/arrow_right_white.svg</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="backspace">
    <property name="geometry">
     <rect>
      <x>265</x>
      <y>35</y>
      <width>50</width>
      <height>40</height>
     </rect>
    </property>
    <property name="icon">
     <iconset resource="icons.qrc">
      <normaloff>:/settings/icons/backspace.svg</normaloff>:/settings/icons/backspace.svg</iconset>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
